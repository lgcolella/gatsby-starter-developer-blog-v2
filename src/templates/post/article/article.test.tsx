/* Vendor imports */
import React from 'react'
import { mount } from 'enzyme'
/* App imports */
import Article from './index'

describe('Post > Article component', () => {
  test('matches snapshot', () => {
    let wrapper = mount(
      <Article html={'<div><h2>Sub title</h2><p>Text paragraph</p></div>'} />
    )

    expect(wrapper).toMatchSnapshot()
  })
})
