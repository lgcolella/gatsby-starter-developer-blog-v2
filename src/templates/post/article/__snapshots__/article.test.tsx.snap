// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Post > Article component matches snapshot 1`] = `
<Article
  html="<div><h2>Sub title</h2><p>Text paragraph</p></div>"
>
  <div
    className="container"
  >
    <article
      dangerouslySetInnerHTML={
        Object {
          "__html": "<div><h2>Sub title</h2><p>Text paragraph</p></div>",
        }
      }
    />
  </div>
</Article>
`;
